### PR TITLE
base quest template

### PR DESCRIPTION
**Pull request by hacktivist123**

### CODE REVIEW

File: tests/profile.test.js [hunk 1]
```diff
@@ -1,36 +0,0 @@
-const puppeteer = require("puppeteer");
```
Comment (limormek) [inline tests/profile.test.js:1]: why did you delete this?

Comment (hacktivist123) [reply tests/profile.test.js:1]: renamed it to design.test.js

**Review by limormek — APPROVED** [review]
